### ISSUE TITLE
5.1.2 (3016) - No images showing in feed items

### ISSUE THREAD

**Issue by slater** [Ranchero-Software/NetNewsWire#2551]
Desktop, 5.1.2 (3016), no feeds are displaying any images.

**Comment by vincode-io** [comment]
Do you have any feeds you can give us as an example?

**Comment by slater** [comment]
https://www.jwz.org/blog/feed/

![Screen Shot 2020-11-03 at 15 31 07](https://user-images.githubusercontent.com/8721/98051881-a2bfdb00-1de9-11eb-9439-eb9a1c942a5a.png)

and all other feeds that have images

**Comment by slater** [comment]
This is only happening since i just upgraded from previous public version

**Comment by vincode-io** [comment]
Are you using a syncing service like Feedly or Feedbin?

**Comment by slater** [comment]
No, this is straight from the site's RSS feed

**Comment by vincode-io** [comment]
So you are using the On My Mac account, correct?

**Comment by slater** [comment]
Correct

**Comment by vincode-io** [comment]
I'm unable to recreate your problem.  That's not good for me, because I have no way to debug it.  Unfortunately we might be have to try some last resort measures to get you back working again.  

First, let's delete NetNewsWire and download a new copy to install.

https://github.com/Ranchero-Software/NetNewsWire/releases/download/mac-5.1.2/NetNewsWire5.1.2.zip

Once that is done, please restart your computer.  This works best if you have a goat or chicken to sacrifice while doing the restart.

Let me know how it goes and if that doesn't resolve your issue, we'll try to come up with another plan.

**Comment by slater** [comment]
A reboot did it!

No goats, chickens, or even donkeys were harmed.
